media.css

```diff
@@ -0,0 +1,131 @@
+@media (max-width: 1550px) {
+  
+
+  
+  .center__v {
+    
+  }
+}
+
+  
+@media (max-width: 1453px) {
+  .middle__title {
+    right: 0px;
+    top:150px;
+  }
+
+  .content {
+    left: 0;
+    bottom: 0;
+    width: 100% ;
+  }
+
+  .container {
+  max-width: 100%;
+  bottom: 0px;
+  top: 900px;
+  }
+  .underside {
+  bottom: 1000px;
+  left: 0px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)